rostwitter/python/rostwitter/twitter.py

```diff
@@ -1,18 +1,19 @@
 # originally from https://raw.githubusercontent.com/bear/python-twitter/v1.1/twitter.py  # NOQA
 
+import math
 import json as simplejson
 import requests
-from requests_oauthlib import OAuth1
-# https://stackoverflow.com/questions/11914472/stringio-in-python3
 try:
-    from StringIO import StringIO ## for Python 2
-except ImportError:
-    from io import StringIO ## for Python 3
+    from itertools import zip_longest
+except (ImportError, ModuleNotFoundError):
```

Comment:
Sorry `ModuleNotFoundError` may be also only for python3.
```suggestion
except ImportError:
```

Reply:
Thanks!